src/web3/addon/keystore.js

```diff
@@ -68,6 +68,16 @@ module.exports.isAccountLocked = (web3, { address }) => {
   return web3.currentProvider.isAccountLocked(address);
 };
 
+module.exports.sign = (web3, { msg, chainId, address }, cb) => {
```

Comment:
[Questions] As I mentioned above I am concern on calling this method `sign` if the signing isn't correspond to the standards. In case you are sure that it does. Just keep it

Reply:
Actually, I will call it specifically "signAuthToken" so people don't use this function instead of the web3 official ones. It also doesn't support chainId atm given IPFS doesn't have this concept.